### PR TITLE
#905 - Offset for recommendation rendering

### PR DESCRIPTION
**Pull request by wpmp**
**What's in the PR**
* adapt changes from webanno/webanno#1212
* fix bug where span annotation texts contained ` and were not escaped

**How to test manually**
* Upstream changes from webanno/webanno#1212 required
* Open a PDF in PDF editor
* Trigger rendering for recommendations (e.g. by opening sidebar)
* See that the whole document is filled with recommendations

**Automatic testing**
* [ ] PR includes unit tests

**Documentation**
* [ ] PR updates documentation

### CODE REVIEW

**Comment by reckart** [comment]
@wpmp could you please have a look at the conflict?

File: inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/RecommendationEditorExtension.java [hunk 1]
```diff
@@ -242,7 +242,12 @@ private void actionRejectRecommendation(AnnotationActionHandler aActionHandler,
     }
     
     @Override
+<<<<<<< HEAD
```
Comment (reckart) [inline inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/RecommendationEditorExtension.java]: Unresolved merge conflict here. The first parameter needs to be `CAS aCas`.

File: inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfanno/model/Span.java [hunk 2]
```diff
@@ -120,7 +120,7 @@ public String toAnnoFileString()
             "page = " + page + "\n" +
             "label = \"" + label + "\"\n" +
```
Comment (reckart) [inline inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfanno/model/Span.java]: It there also a risk that the label contains special characters that need to be escaped (cf. text)?

File: inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/RecommendationEditorExtension.java (hunk 1 above)
Comment (wpmp) [reply inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/RecommendationEditorExtension.java]: Oh no, sorry, I currently cannot compile the project, so I didn't notice it...

File: inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfanno/model/Span.java (hunk 2 above)
Comment (wpmp) [reply inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfanno/model/Span.java]: I think currently not, however definitely worth escaping it nevertheless.
Will fix this.

**Comment by reckart** [comment]
This would be the next one for me to merge if it builded.

**Comment by wpmp** [comment]
Sorry, it should finally build now ...

**Comment by wpmp** [comment]
@reckart Do you know where `Rate limit exceeded` in KB module comes from?

**Comment by reckart** [comment]
I guess we are abusing the Wikidata SPARQL endpoint a bit too much.

**Comment by reckart** [comment]
Is there a reason why you always force-push into this branch?

**Comment by reckart** [comment]
Jenkins, can you test this please?

**Comment by wpmp** [comment]
Oh, ok.

For the commits I had the wrong author email set and I changed that.
And the errors that prevented proper building were from merging so I thought it would fit the last commit, so I used `commit --amend`.
Are those valid reasons or should I not do it like that?

**Comment by reckart** [comment]
Good reasons - it was just that there were so many force pushes that I was wondering ;)

**Comment by wpmp** [comment]
Ok :)

**Pull request merged**